ftw/simplelayout/contenttypes/browser/templates/textblock.pt

```diff
@@ -15,6 +16,7 @@
   <tal:image tal:condition="image">
     <div tal:attributes="class image/wrapper_css_classes">
         <div class="imageContainer">
+            <div metal:use-macro="view/context/@@ftw.simplelayout.images.macros/low_image_quality_indicator"></div>
```

Comment:
Why do we need to have a macro? It does not seem to be reused.
Could we move the tags from the macro here and get rid of the macro?

Reply:
We'll use this markup for the galleryblock too... in the future, we'll use it for further implementations.

Do you think, it's useless?